libs/cli/src/lib/index.ts

```diff
@@ -3,69 +3,22 @@ import * as process from 'node:process';
 import { program } from 'commander';
 
 import packageJson from '../../package.json';
-import { createApp, createVersion, getApp, getAppVersions, markVersionAsBuilt } from './api';
-import { validateEnv, VerityCiContextKeys } from './env';
+import { validateEnv } from './env';
+import { upsertVersion } from './upsert-version';
 
 program.version(packageJson.version).description('CLI tool for Verity CI integration');
 
-program
-  .command('run')
-  .description('Run a Verity CI job')
-  .action(async () => {
-    await run();
+const versionsCommand = program.command('version').description('Manage versions');
+
+versionsCommand
+  .command('upsert')
+  .description(
+    'Mark the current version as built, or create and mark a new version if it does not exist.',
```

Comment:
description should the third-person singular subject. (Marks the current version)